web/html/src/manager/audit/cveaudit/cveaudit.tsx

```diff
@@ -219,6 +228,30 @@ class CVEAudit extends React.Component<Props, State> {
     });
   };
 
+  getPatchStatusAccuracyWarning = (row) => {
+    const dataSources: string[] = row.scanDataSources;
+    if (!dataSources) {
+      Loggerhead.error("CVE audit data sources were not supplied by server.");
+      return t("Error, see console");
```

Comment:
I think here and below, the errors we log are also fine as tooltips, so `return t("CVE audit data sources were not supplied by server")` and `return t("Invalid scan data sources")` work imo. But this is a minor nitpick.